nglview/show.py

```diff
@@ -337,6 +337,14 @@ def show_rdkit(rdkit_mol, **kwargs):
     ----------
     rdkit_mol : rdkit.Chem.rdchem.Mol
     kwargs : additional keyword argument
```

Comment:
Can you please add a test with new options to make sure the code is excised? 

https://github.com/arose/nglview/blob/4dd648de75f5f0e769651d725d2491b168bdec11/nglview/tests/test_widget.py#L584